spam-villa/testnet-keys.env

```diff
@@ -0,0 +1 @@
+ETHEREUM_PRIVATE_KEYS=a,b,c,d,e,f
```

Comment:
File is marked as ignore but included.  Is this intentional so that people filling in keys won't accidentally commit them, but the file template remains?

Reply:
exactly